airflow/gcp/operators/dataproc.py

```diff
@@ -485,7 +485,20 @@ def _usable_existing_cluster_present(self, service):
 
             if existing_status == 'RUNNING':
                 self.log.info('Cluster exists and is already running. Using it.')
-                return True
+                return existing_cluster
```

Comment:
I have never looked into this operator previously, but one point that I would like to point out:
Earlier this method is return `True`/`False`, and after your change it will return mixed things (`False`/result from DataProcHook.find_cluster(), which can be an actual value or `None`). I do not find this a good practice.

Reply:
I agree. I did this because it's a restricted method used only in execute where I handle this output. And that's better than failing operator (system tested).

Probably I will close this issue soon because I am refactoring Dataproc hook / operators.

Reply:
I agree it's strange to do it this way. It should be consistently one type. Waiting for the refactor :)